src/apps/Permissions/AppCard.js

```diff
@@ -1,100 +1,95 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import { Text, Card, Badge, theme, unselectable } from '@aragon/ui'
+import { Card, Badge, unselectable, GU } from '@aragon/ui'
 import { AppType } from '../../prop-types'
 import { shortenAddress } from '../../web3-utils'
 import AppIcon from '../../components/AppIcon/AppIcon'
 
-class AppCard extends React.PureComponent {
-  static propTypes = {
-    app: AppType.isRequired,
-    onOpen: PropTypes.func.isRequired,
-  }
+const AppBadge = Badge.App
 
-  handleClick = () => {
-    this.props.onOpen(this.props.app.proxyAddress)
-  }
-  render() {
-    const { app } = this.props
-    const {
-      name,
-      identifier,
-      isAragonOsInternalApp,
-      hasWebApp,
-      proxyAddress,
-    } = app
+const AppCard = React.memo(function AppCard({ onOpen, app }) {
+  const {
+    name,
+    identifier,
+    isAragonOsInternalApp,
+    hasWebApp,
+    proxyAddress,
+  } = app
 
-    const instanceTitle = `Address: ${proxyAddress}`
-    const instanceLabel = isAragonOsInternalApp
-      ? 'System App'
-      : !hasWebApp
-      ? 'Background App'
-      : identifier || shortenAddress(proxyAddress)
+  const onClick = useCallback(() => {
+    console.log('open', proxyAddress)
```

Comment:
Do we want this here? If so, should it be only the `log` function we use elsewhere?

Reply:
Fixed in 424968e.